custom_components/xiaomi_miio_fan/fan.py

```diff
@@ -618,6 +627,14 @@ async def async_set_oscillation_angle(self, angle: int) -> None:
             "Setting angle of the miio device failed.", self._device.set_angle, angle
         )
 
+    async def async_set_delay_off(self, delay_off_countdown: int) -> None:
+        """Set scheduled off timer in seconds"""
```

Comment:
@syssi @insajd You guys might want to update the function docstring to minutes since it's now in minutes

Reply:
You're right, it's minutes here. I have updated this in my branch. Do I do separate PR for this?

Reply:
Just push a new commit.